09-Multiparticle-Collision-Dynamics/01-Pressure-Driven-Flow.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>This section jumps straight to setting all of the needed methods and forces - but these have not yet been introduced.</p><p> </p><p>Add a paragraph at the beginning of this section that briefly explains what the <code>mpcd.Integrator</code> does and list the methods it applies (streaming_method, collision_method, solvent_sorter, mpcd_particle_body_force). It is also unclear what the triggers are or how they are used. Please add pseudocode (like that I wrote for Simulation.run: <a href="https://hoomd-blue.readthedocs.io/en/v4.7.0/package-hoomd.html#hoomd.Simulation.run" target="_blank">https://hoomd-blue.readthedocs.io/en/v4.7.0/package-hoomd.html#hoomd.Simulation.run</a>) that outlines the mpcd integration logic.</p><p> </p><p>Also, why is it "solvent" sorter but "mpcd_particle" body force? Are these referring to two different things? If not, perhaps we should modify the MPCD code in HOOMD before release and give them the same name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
You are right, I thought I had grepped and replaced all the uses of solvent in the API, but I must have missed this. It should be `mpcd_particle_sorter`. I will open an issue and PR to fix before release

Reply:
<p>It might also be helpful to add this pseudocode in the <code>Integrator</code> documentation itself. </p>

Reply:
<p>Sorry, following up on this: I was mistaken, and I did make this correction in the code. This is a typo in the tutorial.</p>

Reply:
Good idea. The psedocode belongs in the documentation and the tutorial should give an overview of the steps.